libr/core/linux_heap_glibc.c

```diff
@@ -355,6 +413,16 @@ static bool GH(r_resolve_main_arena)(RCore *core, GHT *m_arena) {
 	if (!ta) {
 		return false;
 	}
+
+	if (main_arena_sym) {
+		GH (update_main_arena)
```

Comment:
uhm. the line below are the arguments for the function call in here. merge those 2 lines